doc/nrf/releases/release-notes-latest.rst

```diff
@@ -197,34 +321,108 @@ The following list summarizes the most important changes inherited from upstream
+    * Fixed a ``getaddrinfo`` resource leak.
+
+* Settings:
+
+  * Updated the implementation to return an error rather than faulting if there is an attempt to read a setting from a channel that does not support reading.
+  * Disallowed modifying the content of a static subtree name.
+
+* LVGL:
+
+  * Updated the library to the new major release v7.0.2.
+  * Aligned LVGL Kconfig constants with suggested defaults from upstream.
+
+* Tracing:
+
+  * Updated the API to check if the init function exists prior to calling it.
+
+* Logging:
+
+  * Fixed immediate logging with multiple backends.
+  * Switched logging thread to use the kernel stack.
+  * Allowed users to disable all shell backends at once using :option:`CONFIG_SHELL_LOG_BACKEND`.
+  * Added a logging backend for the Spinel protocol.
+  * Fixed timestamp calculation when using NEWLIB.
+
+* Shell:
+
+  * Switched to use the kernel stack.
+  * Fixed the select command.
+  * Fixed prompting dynamic commands.
+
+* libc:
+
+  * Simplified newlib malloc arena definition.
 
 * Devicetree:
 
   * Removed all nRF-specific aliases to particular hardware peripherals, because they are no longer needed now that nodes can be addressed by node labels.
     For example, you should now use ``DT_NODELABEL(i2c0)`` instead of ``DT_ALIAS(i2c_0)``.
+    If you use something similar to ``source $(SOC_DIR)/<path>``, change it to ``rsource <relative>/<path>`` or similar.
+
+  * BOARD, SOC, DTS, and ARCH roots can now be specified in each module's :file:`zephyr/module.yml` file (see :ref:`modules_build_settings`).
 
 * Build system:
 
   * Renamed the ``TEXT_SECTION_OFFSET`` symbol to ``ROM_START_OFFSET``.
   * Added a number of iterable section macros to the set of linker macros, including ``Z_ITERABLE_SECTION_ROM`` and ``Z_ITERABLE_SECTION_RAM``.
   * Added a new Zephyr Build Configuration package with support for specific build configuration for Zephyr derivatives (including forks).
     See :ref:`zephyr:cmake_pkg` for more information.
+  * Removed the set of ``*_if_kconfig()`` CMake functions.
+    Use ``_ifdef(CONFIG_ ...)`` instead.
 
 * Samples:
 
   * Updated the :ref:`zephyr:nrf-system-off-sample` to better support low-power states of Nordic Semiconductor devices.
   * Updated the :ref:`zephyr:usb_mass` to perform all application-level configuration before the USB subsystem starts.
     The sample now also supports FAT file systems on external storage.
+  * Updated the :ref:`zephyr:nvs-sample` sample to do a full chip erase when flashing.
+  * Fixed the build of the :ref:`zephyr:bluetooth-mesh-onoff-level-lighting-vnd-sample` application with mcumgr.
+  * Added new commands ``write_unaligned`` and ``write_pattern`` to the :ref:`zephyr:samples_flash_shell`.
+  * Fixed the ``cmd_hdr`` and ``acl_hdr`` usage in the :ref:`zephyr:bluetooth-hci-spi-sample` sample.
+  * Removed the NFC sample.
+  * Fixed an uninitialized ``spi_cfg`` in the :ref:`zephyr:i2c_fujitsu_fram` sample.
```

Comment:
Is this the correct sample? And do we need really need this line in the highlights for NCS?

Reply:
removed